### PR TITLE
[artifactory] Make ingress path configurable

### PR DESCRIPTION
**Pull request by stevie-**
Depending on ingress used, path is interpreted as / or wildcard /*.

#### PR Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] CHANGELOG.md updated
- [x] Variables and other changes are documented in the README.md

**What this PR does / why we need it**:
GCE ingress or AWS ALB ingress interpret ingress path differently as f.e. nginx ingress.

### CODE REVIEW

**Comment by stevie-** [comment]
No idea why helm install times out in Build test.

**Comment by eldada** [comment]
@stevie- it might be related to Bintray issues, since our images are pulled from there. I initiated a new build.

**Comment by stevie-** [comment]
@eldada thanks

File: stable/artifactory-ha/templates/ingress.yaml [hunk 1]
```diff
@@ -29,7 +29,7 @@ spec:
   - host: {{ $host | quote }}
     http:
       paths:
-        - path: /
+        - path: {{ .Values.ingress.path }}
```
Comment (jaspermarcus) [inline stable/artifactory-ha/templates/ingress.yaml]: Should this have been `$.Values.ingress.path` instead? I'm trying to upgrade to `0.11.14` and have ran into an issue around this templating. As I understand it, the `.` no longer points to the root context inside a range function (https://github.com/helm/helm/issues/1311#issuecomment-399577623)

Comment (eldada) [reply stable/artifactory-ha/templates/ingress.yaml]: @stevie- can you verify this?

Comment (eldada) [reply stable/artifactory-ha/templates/ingress.yaml]: @jaspermarcus - we are fixing it with https://github.com/jfrog/charts/pull/265.

Comment (stevie-) [reply stable/artifactory-ha/templates/ingress.yaml]: Ahh, Thanks, missed that. Guess tests coverage needs to be udpated as well.

Comment (jaspermarcus) [reply stable/artifactory-ha/templates/ingress.yaml]: Thanks for the quick response! 👍

**Pull request merged**